src/views/ProductPage.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div class="productPage">
+		<Header />
+		<div class="main">
+			<v-container class="lighten-5">
+				<v-row no-gutters>
+					<v-breadcrumbs :items="productRoute" dark />
+				</v-row>
+				<v-row>
+					<h1>
+						{{ activeProduct.name }}
+					</h1>
+				</v-row>
+				<v-row class="mat-5">
+					<v-col cols="12" sm="4">
+						<v-carousel hide-delimiters :show-arrows="false">
+							<v-carousel-item :src="activeProduct.image_url" />
+						</v-carousel>
+					</v-col>
+					<v-col cols="12" sm="8">
+						<v-card elevation="0">
+							<v-card-title class="pa-0">$170</v-card-title>
+							<v-btn color="green" elevation="0" @click="buy_btnHandler">
+								{{ $t("product.buy_btn") }}
+							</v-btn>
+						</v-card>
+
+						<v-tabs
+							v-model="tab"
+							background-color="transparent"
+							color="basil"
+							grow
+						>
+							<v-tab v-for="item in items" :key="item">
+								{{ $t(`product.${item.toLowerCase()}`) }}
+							</v-tab>
+						</v-tabs>
+
+						<v-tabs-items v-model="tab">
+							<v-tab-item>
+								<v-card color="basil" flat>
+									<DescriptionList
+										class="mmat-2"
+										:list="productDescriptionObj"
+									/>
+								</v-card>
+							</v-tab-item>
+							<v-tab-item>
+								<v-card color="basil" flat>
+									<p>{{ activeProduct.description }}</p>
+									<p>{{ activeProduct.brewers_tips }}</p>
+									<p>
+										{{ $t("product.contributed_by") }}
+										{{ activeProduct.contributed_by }}
+									</p>
+								</v-card>
+							</v-tab-item>
+						</v-tabs-items>
+
+						<!-- <h3 class="mmat-5">{{ $t("product.characteristics") }}</h3>
+						<DescriptionList class="mmat-2" :list="productDescriptionObj" /> -->
+					</v-col>
+				</v-row>
+			</v-container>
+		</div>
+	</div>
+</template>
+
+<script>
+import Header from "@/components/Header"
+import { mapActions, mapState } from "vuex"
+import routeNames from "@/router/routeNames"
+import DescriptionList from "@/components/DescriptionList"
+export default {
+	name: "ProductPage",
+	components: {
+		Header,
+		DescriptionList,
+	},
+	data() {
+		return {
+			tab: null,
+			items: ["Characteristics", "Description"],
+		}
+	},
+	computed: {
+		...mapState("products", ["activeProduct"]),
+		productRoute() {
+			return [
+				{
+					text: this.$t("menu.home"),
+					exact: true,
+					disabled: false,
+					to: { name: routeNames.home },
+				},
+				{
+					text: this.activeProduct.name,
+					disabled: true,
+					to: { name: routeNames.product, params: this.$route.params.id },
+				},
+			]
+		},
+		productDescriptionObj() {
+			let objCopy = Object.assign({}, this.activeProduct)
+			objCopy = this.spliceOnObj(objCopy, 6, 8)
+			return objCopy
+		},
```

Comment:
I don't understand what are you doing here could you rework logic in a more appropriate way?

Reply:
I write part of props from activeProduct object to new one

Reply:
do not find a better way to do the same